src/redux/todoSlice.js

```diff
@@ -0,0 +1,9 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const TodoSlice = createSlice({
+    name: "todos",
+    initialState: [],
+    reducers: {},
+});
+
+export default TodoSlice;
```

Comment:
if you export TodoSlice.reducer it will fix your "Store does not have a valid reducer. Make sure the argument passed to combineReducers is an object whose values are reducers." error